### ISSUE TITLE
Pane Comparison

### ISSUE THREAD

**Issue by Horizon82** [files-community/Files#14375]
### What feature or improvement do you think would benefit Files?

Add a "midsection" between the two panes for common tasks like copy/move/compare.  This is another way to perform common tasks without the need for multiple clicks/button presses.  Often times copy/move to a full directory means accidentally moving to an unintended folder whether that be via drag and drop or a traditional copy/paste.  This often occurs due to lack of empty real estate in a given folder.

### Requirements

Splitting up the two panes to facilitate a "midsection"
"midsection" needs to be pane aware to move files in the proper direction.

### Files Version

3.0.15.0

### Windows Version

11 Pro 23H2

### Comments

![image](https://github.com/files-community/Files/assets/155760601/9dcdce6d-82f6-4425-a6fb-85ea13143716)

Main splitter "midsection" of FreeCommander

**Comment by yaira2** [comment]
Thanks for the feedback, this is certainly something I can see us doing. I don't use dual pane on a regular basis so hearing your input is very helpful.
1. Which features do you see being added to this toolbar?
2. We're tracking a request to support for than two panes, how do you see this working together with three or four panes?

**Comment by Horizon82** [comment]
I've used the copy/move feature a good deal so those are my specific recommendations.  I haven't found a use case for the compare feature though I imagine there is a power user out there that would find it useful.  I suppose the recycle bin could be helpful.

Initial thought for 3+ panes is when you click on the middle button that a drop down pops up and you select the intended pane (or all panes). The only other way it would work is if each section between panes had their own split section. It would certainly increase the real estate required by Files but I imagine anyone using more than 2 has an ultrawide and space isn't an issue.

**Comment by nananan19181** [comment]
I agree on the dual pane having a toolbar with ability to copy/move items back and forth as a button would be super helpful.

In addition I would like to see the address (path) bar separately for both panes.  I've caught myself a few times getting confused on active pane and what I'm looking at.

Also with those features adding 3-5 panes could become possible.  I move around a lot of files between "environments" on network drives so test, qa, prod, disaster recovery would all be panes and then copying/moving files between them by hitting a button vs dragging and dropping would be awesome.
